source/Nevermore/RelationalStoreConfiguration.cs

```diff
@@ -50,6 +50,8 @@ public RelationalStoreConfiguration(Func<string> connectionStringFunc)
 
             QueryLogger = new DefaultQueryLogger();
 
+            RelatedDocumentsGlobalTempTableNameGenerator = () => Guid.NewGuid().ToString().Replace('-', '_');
```

Comment:
You could use Guid.ToString("N") to remove separator characters.

Reply:
TIL `Guid.ToString()` has its own options ;)